docs/source/support/frequently-asked-questions.rst

```diff
@@ -583,9 +583,9 @@ After this step, the reusable workflow then runs the ``pytest`` command. To see
 I have a private repo and don't have a Codecov paid plan.  Can I modify the CI workflows for this situation?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-By default, the workflows in ``tests-on-pr.yml`` and ``matrix-and-codecov-on-merge-to-main.yml`` will have a section that uploads the code coverage to Codecov, and CI will fail if unsuccessful. If you have a private repository and Codecov cannot be run, CI will always fail, and this is undesirable.
+By default, the workflows in ``tests-on-pr.yml`` and ``matrix-and-codecov-on-merge-to-main.yml`` will have a section that uploads the code coverage to Codecov, and CI will fail if unsuccessful. Codecov is available for free on public repositories but requires a paid plan to run on private repositories. In this latter case, CI will always fail because Codecov fails. ``scikit-package`` has alternative CI workflows for this situation.
```

Comment:
"By default, the workflows in ``tests-on-pr.yml`` and ``matrix-and-codecov-on-merge-to-main.yml`` will have a section that uploads the code coverage to Codecov, and CI will fail if unsuccessful. Codecov is available for free on public repositories but requires a paid plan to run on private repositories. In this latter case.  If you would like to turn off Codecov we offer an alternative CI workflow that runs the tests without Codecov."